xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-table/plugin.js

```diff
@@ -69,6 +69,16 @@
     ['txtBorder', 'txtWidth', 'txtCellSpace', 'txtCellPad'].forEach(function(fieldId) {
       delete infoTab.get(fieldId)['default'];
     });
+    // Set a new default for the Headers value
+    infoTab.get( 'selHeaders' )[ 'default' ] = 'row';
+    // Create a warning message for the accessibility without headers
```

Comment:
```suggestion
    // Create a warning message for the accessibility without headers.
```

If it's a phrase then it should end with a dot IMO :) .